src/viam_mlmodelservice_triton_impl.cpp

```diff
@@ -419,18 +444,55 @@ class Service : public vsdk::MLModelService, public vsdk::Stoppable, public vsdk
         }
         state->model_name = std::move(*model_name_string);
 
-        auto model_version = attributes->find("model_version");
-        if (model_version != attributes->end()) {
-            auto* const model_version_value = model_version->second->get<double>();
-            if (!model_version_value || (*model_version_value < 1) ||
-                (std::nearbyint(*model_version_value) != *model_version_value)) {
+        // Pull the model repository path out of the configuration.
```

Comment:
I don't want to delay this further, but I think it'd be good to add a TODO here to add logic that errors out if you try to set *both* `model_repository_path` *and* `model_path`, because I think it is one or the other.

Reply:
Great idea! I'll put in a TODO and address in a future PR.